src/index.html

```diff
@@ -3,10 +3,442 @@
+              src="./images/imac.jpeg"
+              alt="Imac"
+            >
+          </div>
+
+          <h2 class="card__heading">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          <p class="card__text-small">
+            Product code: 195434
+          </p>
+          <div class="reviews">
+            <div class="star stars--4">
+              <div class="star__stars"></div>
+              <div class="star__stars"></div>
+              <div class="star__stars"></div>
+              <div class="star__stars"></div>
+              <div class="star__stars"></div>
+            </div>
+            <div class="reviews__total">
+              Reviews: 5
+            </div>
+          </div>
+          <div class="card__price">
+            <div class="card__price-text">
+              Price:
+            </div>
+            <div class="card__price-cost">
+              $2,199
+            </div>
+          </div>
+          <div>
+            <a
+              href="#"
+              class="card__buy"
+              data-qa="card-hover"
+            >
+              buy
+            </a>
+          </div>
+        </div>
+        <div class="card catalog__card">
+          <div class="card__image">
+
+            <img
+              class="card__image image"
+              src="./images/imac.jpeg"
+              alt="Imac"
+            >
+          </div>
+
+          <h2 class="card__heading">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          <p class="card__text-small">
+            Product code: 195434
+          </p>
+          <div class="reviews">
+            <div class="star stars--4">
+              <div class="star__stars"></div>
+              <div class="star__stars"></div>
+              <div class="star__stars"></div>
+              <div class="star__stars"></div>
+              <div class="star__stars"></div>
+            </div>
+            <div class="reviews__total">
+              Reviews: 5
+            </div>
+          </div>
+          <div class="card__price">
+            <div class="card__price-text">
+              Price:
+            </div>
+            <div class="card__price-cost">
+              $2,199
+            </div>
+          </div>
+          <div>
+            <a
+              href="#"
+              class="card__buy"
+              data-qa="card-hover"
+            >
+              buy
+            </a>
+          </div>
+        </div>
+        <div class="card catalog__card">
+          <div class="card__image">
+            <img
+              class="card__image image"
+              src="./images/imac.jpeg"
+              alt="Imac"
+            >
+          </div>
+          <h2 class="card__heading">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          <p class="card__text-small">
+            Product code: 195434
+          </p>
+          <div class="reviews">
```

Comment:
I don't think that you will want to reuse stars (which is already a BEM block) with this reviews count with same styles. Make element from it